terraform/environments/mlra/main.tf

```diff
@@ -0,0 +1,49 @@
+module "lb-access-logs-enabled" {
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer"
+  providers = {
+    aws.bucket-replication = aws
+  }
+
+  vpc_all = local.vpc_all
+  #existing_bucket_name               = "my-bucket-name"
+  application_name           = local.application_name
+  public_subnets             = [data.aws_subnet.public_subnets_a.id, data.aws_subnet.public_subnets_b.id, data.aws_subnet.public_subnets_c.id]
+  loadbalancer_egress_rules  = local.loadbalancer_egress_rules
+  loadbalancer_ingress_rules = local.loadbalancer_ingress_rules
+  tags                       = local.tags
+  account_number             = local.environment_management.account_ids[terraform.workspace]
+  region                     = local.application_data.accounts[local.environment].region
+  enable_deletion_protection = false
+  idle_timeout               = 60
+}
+
+locals {
+  loadbalancer_ingress_rules = {
+    "cluster_ec2_lb_ingress" = {
+      description     = "Cluster EC2 loadbalancer ingress rule"
+      from_port       = 32768
+      to_port         = 61000
+      protocol        = "tcp"
+      cidr_blocks     = []
+      security_groups = []
+    },
+    "cluster_c2_bastion_ingress" = {
+      description     = "Cluster EC2 bastion ingress rule"
+      from_port       = 22
+      to_port         = 22
+      protocol        = "tcp"
+      cidr_blocks     = ["10.200.0.0/20"]
```

Comment:
This is likely to be an invalid CIDR range, as this is the CIDR range of a subnet in the LAA environment. Perhaps we can refer to data sources that MP provide in data.tf to get the appropriate CIDR range for the MP VPC we are deploying to?